src/core/server/ui_settings/ui_settings_config.ts

```diff
@@ -53,13 +53,7 @@ const deprecations: ConfigDeprecationProvider = ({ unused, renameFromRoot }) =>
  */
 
 const configSchema = schema.object({
-  overrides: schema.object(
-    {
-      'theme:darkMode': schema.maybe(schema.boolean({ defaultValue: true })),
-      'theme:version': schema.string({ defaultValue: 'v7' }),
-    },
-    { unknowns: 'allow' }
-  ),
+  overrides: schema.object({}, { unknowns: 'allow' }),
```

Comment:
Can you explain this change? If I remember correctly, I think it was added to make sure users who had previously set this the the v8 (beta) theme didn't get old styles that caused conflicts.

Reply:
gotcha so `{}` basically respects whatever value? or does it do some input validation that ensure that it's one or another value?

Reply:
oh nvm. are we sure we want to remove it? I believe this was actually added if any user was actually on `v8 (beta)` this logic was forcefully overriding it.

But tbh, I'm not sure how keen I am on carrying legacy logic over multiple iterations just in case someone is using v8 (beta) then upgrades to where they can deselect that theme.